tests/test_atmos.py

```diff
@@ -1,229 +1,22 @@
-
-    hurs = atmos.relative_humidity(
-        tas=tas,
-        huss=huss,
-        ps=ps,
-        method="sonntag90",
-        ice_thresh="0 degC",
-    )
-    np.testing.assert_allclose(hurs, hurs_exp, atol=0.5, rtol=0.005)
-    assert hurs.name == "hurs"
-
-
-def test_specific_humidity(tas_series, hurs_series, huss_series, ps_series):
-    tas = tas_series(np.array([20, -10, 10, 20, 35, 50, 75, 95]) + K2C)
-    hurs = hurs_series([150, 10, 90, 20, 80, 50, 70, 40, 30])
-    ps = ps_series(1000 * np.array([100] * 4 + [101] * 4))
-    huss_exp = huss_series([np.nan, 1.6e-4, 6.9e-3, 3.0e-3, 2.9e-2, 4.1e-2, 2.1e-1, 5.7e-1])
-
-    huss = atmos.specific_humidity(
-        tas=tas,
-        hurs=hurs,
-        ps=ps,
-        method="sonntag90",
-        ice_thresh="0 degC",
-    )
-    np.testing.assert_allclose(huss, huss_exp, atol=1e-4, rtol=0.05)
-    assert huss.name == "huss"
-
-
-def test_specific_humidity_from_dewpoint(tas_series, ps_series, huss_series):
-    tdps = tas_series([272, 283, 293])
-    ps = ps_series([100000, 105000, 110000])
-    # Computed from MetPy
-    # >>> from metpy.units import units as u
-    # >>> from metpy.calc import specific_humidity_from_dewpoint as sh
-    # >>> sh([100000, 105000, 110000] * u.Pa, [272, 283, 293] * u.degK)
-    # array([0.0035031, 0.00722795, 0.01319614]) < Unit('dimensionless') >
-    huss_exp = huss_series([0.0035031, 0.00722795, 0.01319614])
-
-    huss = atmos.specific_humidity_from_dewpoint(
-        tdps=tdps,
-        ps=ps,
-        method="sonntag90",
-    )
-    np.testing.assert_allclose(huss, huss_exp, atol=1e-4, rtol=0.05)
-    assert huss.name == "huss"
-
-
-@pytest.mark.parametrize("method", ["tetens30", "wmo08", "aerk96", "buck81"])
-def test_dewpoint_from_specific_humidity(atmosds, method):
-    tdps = atmos.dewpoint_from_specific_humidity(huss=atmosds.huss, ps=atmosds.ps)
-    # atmosds.huss was computed with Sonntag90, so we can't expect a perfect match
-    xr.testing.assert_allclose(tdps, atmosds.tdps, atol=0.2, rtol=0.001)
-
-
-def test_snowfall_approximation(pr_series, tasmax_series):
-    pr = pr_series(np.ones(10))
-    tasmax = tasmax_series(np.arange(10) + K2C)
-
-    prsn = atmos.snowfall_approximation(pr, tas=tasmax, thresh="5 degC", method="binary")
-
-    np.testing.assert_allclose(prsn, [1, 1, 1, 1, 1, 0, 0, 0, 0, 0], atol=1e-5, rtol=1e-3)
-
+class TestLateFrostDays:
+    def test_late_frost_days(self, atmosds):
+        tasmin = atmosds.tasmin
 
-def test_rain_approximation(pr_series, tas_series):
-    pr = pr_series(np.ones(10))
-    tas = tas_series(np.arange(10) + K2C)
+        # Expected values
+        exp = [2, 9, 72, 24, 0]
 
-    prlp = atmos.rain_approximation(pr, tas=tas, thresh="5 degC", method="binary")
+        lfd = atmos.late_frost_days(tasmin, date_bounds=("04-01", "06-30"))
 
-    np.testing.assert_allclose(prlp, [0, 0, 0, 0, 0, 1, 1, 1, 1, 1], atol=1e-5, rtol=1e-3)
+        np.testing.assert_allclose(lfd.isel(time=0), exp, rtol=1e-03)
```

Comment:
this is just some test that was already there, but since a lot of stuff is removed, it's highlighted? the implementation has changed a bit?